core/src/main/res/values/strings.xml

```diff
@@ -63,6 +63,9 @@
   <string name="tts_lang_not_supported">The language of this page is not supported. The article may not be properly read.</string>
   <string name="no_reader_application_installed">Could not find an installed application for this type of file</string>
   <string name="no_email_application_installed">Please install an email service provider or email us at %1s</string>
+  <string name="no_browser_application_installed">Please install any web browser to open it e.g. chrome or firefox.</string>
+  <string name="no_pdf_application_installed">Please install any pdf reader application to open it.</string>
+  <string name="no_epub_application_installed">Please install any epub reader application to open it.</string>
```

Comment:
No mime-type specific error. Just something "Please install a reader application to be able to open this content."